tests_new/integration_tests/modules/share_base/fresh_deployment.py

```diff
@@ -0,0 +1,106 @@
+from assertpy import assert_that
+
+from dataall.modules.shares_base.services.shares_enums import ShareItemStatus, ShareObjectStatus
+from tests_new.integration_tests.errors import GqlError
+
+from tests_new.integration_tests.modules.share_base.queries import (
+    submit_share_object,
+    add_share_item,
+    get_share_object,
+    reject_share_object,
+    approve_share_object,
+    revoke_share_items,
+)
+from tests_new.integration_tests.modules.share_base.utils import check_share_ready
+
+
+def test_create_share_object(share1):
+    assert_that(share1.status).is_equal_to(ShareObjectStatus.Draft.value)
```

Comment:
We could also add assertions on the principal of the share and a new test for create_share_object for consumption roles that makes sure that the groupUri and the principal are correct